docs/external-contributions/external-contributions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: external-contributions
+title: External Contributions
+hide_title: true
```

Comment:
Nitpick: In conjunction with #1887, can we do `hide_title: false` here?

Reply:
yep, done